### DISCUSSION TITLE
Exact match with braces and semicolon is somewhat strange

### DISCUSSION THREAD

**Discussion by danlark1** [community/community#9965] [Other Feature Feedback, Questions, & Ideas]
![2022-01-11-003235_813x628_scrot](https://user-images.githubusercontent.com/15173761/148860919-451a8336-2279-4910-9660-95898aaada3d.png)

Even though it finds everything correctly, the message is complaining about ambiguity of the qualifier but I believe all qualifiers are ascii based so it can be removed. Also the highlighted snippets are weird, they match quotation marks

**Comment by colinwm** [comment]
Good point, the warning is definitely not necessary there, I'll tune it.

As for the snippet highlighting, that's a known bug, sorry about that! Working on it.
